git-refresher/pull-requests.qmd

```diff
@@ -0,0 +1,99 @@
+
+## Creating pull requests
+
+Once you have made all the changes you want to include in this batch of work, you'll bundle all of your commits together into a pull request for review.
+
+::: panel-tabset
+## Git Bash
+
+1. Ensure you are on the development branch. 
+
+``` bash
+git checkout -b <github_username>/development
+```
+
+2. Check all your latest changes are pushed to the remote repository. 
+``` bash
+git push origin <github_username>/development
+```
+3. Create your pull request, adding a title and a description of your changes 
+
+``` bash
+gh pr create --base <github_id>/main --head <github_id>/development --title "Update to log book" --body "Updating logbook to reflect completing the development branch task."
+```
+
+## Rstudio and GitHub Website
+
+1. In the Git panel, ensure you are on the development branch. 
+2. Check you have pushed your latest changes to the remote repository by clicking the green 'Push' button in the Git window.
+3. Go to the [git-academy-sandbox](https://github.com/dfe-analytical-services/git-academy-sandbox) repository page on GitHub.
+4. Navigate to the Pull requests tab underneath the repository title. 
+
+![](images/pull_requests_tab.png){fig-alt="Higlighting the Pull requests tab on GitHub."}
+
+
+5. Open a pull request by clicking the "New pull request" button.
+
+6. Ensure the base branch is set to your own main branch (<github_username>/main)  and the compare branch is set to your development branch (<github_username>/development).
+
+![](images/pull_request_branch_selection.png){fig-alt="Higlighting where to change the base and compare branches once you have pressed the 'create a pull request' button on GitHub."}
+
+
+7. Create Pull Request: Click the "Create pull request" button. Add a title and description for your pull request. You may wish to explain what changes and been made, and why. 
+
+8. You don't need to add any reviewers in this training, but please do take a moment to notice where you can add team mates to review your pull requests in your own work to the right of the description. Any reviewers you add will be notified by email that there is a pull request for them to review.  
+
+9. Once you are happy with your title and description, click 'Create pull request' to confirm. 
+
+10. You will then want to check whether there are any merge conflicts. If there are merge conflicts, they will need resolving before you are able to complete your pull request. 
+
+## GitHub Desktop
+
+1. Ensure you are on the development branch. 
+
+2. Click on the 'Branch' menu and 'Create pull request' 
+
+3. Click 'open in browser'. This will open GitHub in your web browser with the pull request form pre-filled. 
+
+4. Add a title and description for you pull request and then click 'Create pull request' 
+
+## VS Code 
+
+## PyCharm 
+::: 
+
+## Reviewing pull requests
```

Comment:
Reckon this should just be a separate page to make it more obvious that it's there.